kubernetes/src/kubernetes/config/kube_config.rb

```diff
@@ -146,7 +146,9 @@ def find_context(name)
         if context['cluster']
           context['cluster'] = find_cluster(context['cluster'])
         end
-        context['user'] = find_user(context['user']) if context['user']
+        if context['user'] && context['user'].length > 0
```

Comment:
Just FYI you can do `context['user'] = find_user(context['user']) if context['user'].blank?`  Then you should be able to keep the line the same.

https://stackoverflow.com/questions/885414/a-concise-explanation-of-nil-v-empty-v-blank-in-ruby-on-rails 

I don't think it matters too much but if you do keep the `if` block can we remove the 2nd if check its kinda redundant and makes it harder to read.

Reply:
Also, I think it would solve the rubocop warnings too :)